.github/workflows/pull-request.yml

```diff
@@ -0,0 +1,29 @@
+name: Pull Request
+
+on:
+  pull_request:
+    paths-ignore:
+      - '*.md'
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: norio-nomura/action-swiftlint@3.2.1
+  test:
```

Comment:
Don't we also need to check the SDK build?

https://github.com/ca-dp/bucketeer-ios-sdk/blob/00078c4698f3758436cd686af0d168535c0acf7c/Makefile#L52-L54

Same for `Build & Integration tests` workflow.

Reply:
Added https://github.com/ca-dp/bucketeer-ios-sdk/pull/96/commits/a02d21ddfa8105ee7671cd5337112ff9aa0b458c https://github.com/ca-dp/bucketeer-ios-sdk/pull/96/commits/c1dc0bb2a7f0ee7891cc0ed0c9a29962d147b568